api/server/router/container/exec.go

```diff
@@ -154,3 +156,27 @@ func (s *containerRouter) postContainerExecResize(ctx context.Context, w http.Re
 
 	return s.backend.ContainerExecResize(vars["name"], height, width)
 }
+
+func (s *containerRouter) postContainerExecKill(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
```

Comment:
should abort `if versions.GreaterThan(version, "1.42")`

Reply:
Do you mean `versions.LessThan(version, "1.42")`?

Reply:
Ah, yes 😅

Reply:
@dtaniwaki I understand this would need to abort if it's called with a daemon that has a version less than 1.42, and I understand that when I do
```go
version := httputils.VersionFromContext(ctx)
```
I get the version of the Docker daemon. So for this new route I must check that the daemon effectively has at least this version to be able to use that internal API call. Am I understanding correctly?

Otherwise please explain why GreaterThan, and what should I write in the error?

What I want to push in the next commit looks like:
```go
func (s *containerRouter) postContainerExecKill(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
	version := httputils.VersionFromContext(ctx)
	if versions.LessThan(version, "1.42") {
		return fmt.Errorf("postContainerExecKill requires API version 1.42, but the Docker daemon API version is %s", version)
	}
```

Reply:
I didn't see the reply from AkihiroSuda when I wrote that. I pushed a commit that fixes it :)